src/features/Sync/SyncOutManager.ts

```diff
@@ -0,0 +1,94 @@
+import { ENTITIES } from "../../common/constants";
+import { SyncLog } from "../../common/services/Database/entities";
+
+const HOSTNAME = '10.0.2.2';
+const PORT = '8080';
+
+const RESOURCES = {
+    COLDCHAIN_V1: 'coldchain/v1'
+}
+
+const PATHS = {
+    LOGIN: 'login',
+    SENSOR: 'sensor', 
+    TEMPERATURE_LOG: 'temperature-log',
+    TEMPERATURE_BREACH: 'temperature-breach'
+}
+class SyncOutManager {
+    private paths: { [key: string]: string };
+
+    constructor() {
+        // TODO: initialise SyncOut manager from dependency container.
+        const rootPath = `http://${HOSTNAME}:${PORT}/${RESOURCES.COLDCHAIN_V1}`;
+
+        const loginPath = `${rootPath}/${PATHS.LOGIN}`;
+        const sensorPath = `${rootPath}/${PATHS.SENSOR}`;
+        const temperatureLogPath = `${rootPath}/${PATHS.TEMPERATURE_LOG}`;
+        const temperatureBreachPath = `${rootPath}/${PATHS.TEMPERATURE_BREACH}`;
+
+        this.paths = {
+            login: loginPath,
+            sensor: sensorPath,
+            temperatureLog: temperatureLogPath,
+            temperatureBreach: temperatureBreachPath,
+        };
+    }
+
+    
+
+    pushLogs = async(logs: SyncLog[]) => {
+        const loginResponse = await fetch(this.paths.login, {
+            method: 'POST',
+            headers: {
+                'content-type': 'application/json',
+            },
+            body: JSON.stringify({
+                username: 'GEN',
+                password: 'secret'
+            })
+        })
+
+        const cookie = loginResponse.headers.get('set-cookie');
```

Comment:
Ideally won't have to get `cookie` for every sync. Can just save this in `DevSlice` state and optionally check for expiry (expiration check would require some changes to mSupply to make cookies expire!).

Reply:
I suggest doing

```
yarn add axios
```

then you can just do `axios.post(this.paths.login, body: {username, password}, {withCredentials: true})`

From that point simply `axios.post(this.paths.X, XSyncLogs, {withCredentials: true} )

to handle cookies (should work haven't tested in RN but works in a browser!)